book/modules/appendix3-exercises.tex

```diff
@@ -38,15 +56,30 @@
 
 			\item $\Set*{\mat{4 \\ 2 \\ 4},\mat{4 \\ 2 \\ 0},\mat{2 \\ 1 \\ 6}}$
 		\end{enumerate}
+        \begin{solution}
+        \begin{enumerate}
+          \item Since $\det(\mat{1 & 1 \\ -3 & 2})=5>0$, the ordered set $\Set*{\mat{1 \\ -3},\mat{1 \\ 2}}$ is a right-handed basis.
+          \item Since $\det(\mat{1 & -2 \\ -3 & 6})=0$, the ordered set $\Set*{\mat{1 \\ -3},\mat{-2 \\ 6}}$ is not a basis.
+          \item Since $\det(\mat{1 & 1 & 1 \\ 0 & 2 & -1 \\ 3 & 5 & 1})=-2$, the ordered set $\Set*{\mat{1 \\ 0 \\ 3},\mat{1 \\ 2 \\ 5},\mat{1 \\ -1 \\ 1}}$ is a left-handed basis.
+          \item Since $\det(\mat{1 & 1 & 1 \\ 4 & 2 & 1 \\ 9 & 3 & 1})=-2<0$, the ordered set $\Set*{\mat{1 \\ 4 \\ 9},\mat{1 \\ 2 \\ 3},\mat{1 \\ 1 \\ 1}}$ is a left-handed basis.
+          \item Since $\det(\mat{4 & 4 & 2 \\ 2 & 2 & 1 \\ 4 & 0 & 6})=0$, the ordered set $\Set*{\mat{4 \\ 2 \\ 4},\mat{4 \\ 2 \\ 0},\mat{2 \\ 1 \\ 6}}$ is not a basis.
+        \end{enumerate}
+        \end{solution}
 
 		\prob Find all values of $a,b\in \R$ so that the set
 			$\Set*{\mat{a^2 \\ ab},\mat{ab \\ b^2}}$ is (a)
 			a right-handed basis, (b) a left-handed basis, (c) not a basis.
+        \begin{solution}
+        Note that $\det(\mat{a^2 & ab \\ ab & b^2})=a^2b^2-a^2b^2=0$. Thus, the set $\Set*{\mat{a^2 \\ ab},\mat{ab \\ b^2}}$ can never be linearly independent. Thus, $\Set*{\mat{a^2 \\ ab},\mat{ab \\ b^2}}$ is not a basis for any $a,b\in\R$.
+        \end{solution}
 
 		\prob Let $M=\mat{a & b \\ c & d}$. The \emph{adjugate matrix} (sometimes called the
 		\emph{classical adjoint}) of $M$,
 		notated $M^{\text{adj}}$, is the matrix given by $M^{\text{adj}}=\mat{d & -b \\ -c & a}$.
 		Prove that if $M$ is invertible, then $\displaystyle M^{-1}=\frac{M^{\text{adj}}}{\det(M)}$.
+        \begin{solution}
+        Since $\displaystyle \frac{M^{\text{adj}}}{\det(M)}M=\mat{\frac{da-bc}{ad-bc} & \frac{db-bd}{ad-bc} \\ \frac{-ca+ac}{ad-bc}& \frac{-cb+ad}{ad-bc}}=\mat{1 & 0 \\ 0 & 1}=I_{2\times 2}$ and $\displaystyle M\frac{M^{\text{adj}}}{\det(M)}=\mat{\frac{ad-bc}{ad-bc} & \frac{-ab+ba}{ad-bc} \\ \frac{cd-dc}{ad-bc}& \frac{-cb+da}{ad-bc}}=\mat{1 & 0 \\ 0 & 1}=I_{2\times 2}$, we conclude that \(\displaystyle M^{-1}=\frac{M^{\text{adj}}}{\det(M)}\).
```

Comment:
Keep the `1/(ad-bc)` out of the matrix until the very last step where everything cancels. It will be much easier to read that way.